packages/cli/src/commands/init/init.ts

```diff
@@ -196,6 +203,7 @@ async function createFromTemplate({
         const installPodsValue = String(installCocoaPods);
```

Comment:
you already have this variable in `didInstallPods` (line125)

Reply:
As discussed here: https://github.com/react-native-community/cli/pull/2204#discussion_r1425289922 

`didInstallPods` is used for a slightly different purpose (also to check if the user didn't specify his choice) and mixing those two made the code more complex.